source/js/components/news/news.jsx

```diff
@@ -46,7 +46,10 @@ class News extends Component {
   }
 
   render() {
+    const currentLanguage = getCurrentLanguage();
```

Comment:
Hi @TheoChevalier! Thank you very much for taking a look at this PR, I was wondering if I can get some feedack on whether or not there is a better way to localize the publish date here? 

What I am doing is getting the locale using the getCurrentLanguage function, and then sending it to [.getLocaleDateString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) in order to return a date in the format "January 2022". If the toLocaleDateString cannot find a matching locale, it defaults to the users preferred locale.

Reply:
Thanks @danielfmiranda! I think that’s the best way of doing it, yup